snapshot/src/test/java/io/zeebe/snapshots/impl/SnapshotChunkReaderTest.java

```diff
@@ -136,78 +125,44 @@ public void shouldSeekToChunk() throws Exception {
     assertThat(snapshotChunks).hasSize(2);
```

Comment:
This and the assertion above can be removed imo - the first one because it's implicitly covered by the asserting that snapshotChunkIds contains exactly 2 chunks ("file1" and "file2"), and the second because we're testing elsewhere that if I read chunk with ids/names "a" and "b", then I will have the right contents for them.